src/completed-list/completed-list.scss

```diff
@@ -229,4 +229,18 @@ title {
   @include type.type-style('heading-compact-02');
   color: $text-02;
   margin-bottom: 0.5rem;
+}
+
+.single-line-date {
+  display: inline-block;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.single-line-action {
+  display: inline-block;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
This is duplication. You can change the class name to something more generic and reuse it in both scenarios

Reply:
Thanks for reviews

Reply:
@sherrif10 please update the PR description with the latest screenshot

Reply:
@pirupius  i added the lasted screen shot.

Reply:
@sherrif10 your screenshot shows the date breaking to the next line. The date should be on a single line.

Reply:
@sherrif10 You can use something like this in the class you added

```
white-space: nowrap;
```